cleverhans_tutorials/mnist_tutorial_tfe.py

```diff
@@ -0,0 +1,253 @@
+"""
+This tutorial shows how to generate adversarial examples using FGSM
+and train a model using adversarial training with TensorFlow Eager.
+
+It is similar to mnist_tutorial_tf.py.
+mnist_tutorial_tf - dynaminc eager execution.
+mnist_tutorial_tf - graph based execution.
+The original paper can be found at:
+https://arxiv.org/abs/1412.6572
+"""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import numpy as np
+import logging
+import sys
+import os
+import tensorflow as tf
+from tensorflow.python.platform import flags
+from distutils.version import LooseVersion
+
+try:
+    tf.enable_eager_execution()
+except AttributeError:
+    if LooseVersion(tf.__version__) < LooseVersion('1.8.0'):
+        error = ('For eager execution',
+                 'use Tensorflow version greather than 1.8.0.')
+        print(error)
+        exit(1)
+
+from cleverhans.utils_mnist import data_mnist
+from cleverhans.utils import AccuracyReport, set_log_level
+from cleverhans.utils_tfe import train, model_eval
+from cleverhans.attacks_tfe import FastGradientMethodTFE
+from cleverhans_tutorials.tutorial_models_tfe import ModelBasicCNNTFE
+
+if tf.executing_eagerly() is True:
+    print('TF Eager Activated.')
+else:
+    error = 'Error Enabling Eager Execution.'
+    raise Exception(error)
+tfe = tf.contrib.eager
+
+FLAGS = flags.FLAGS
+
+
+# Keeps track of implemented attacks.
+# Maps attack string taken from bash to attack class
+# -- 'fgsm' : FastGradientMethod
+
+ListOfAttacks = {
```

Comment:
1. It is confusing to call a dictionary ListOfAttacks.
2. It's a global constant, so it should be all uppercase.

How about AVAILABLE_ATTACKS?